themis-client/src/main/java/org/apache/hadoop/hbase/themis/mapreduce/ThemisTableRecordReaderImpl.java

```diff
@@ -205,20 +209,18 @@ private void updateCounters() throws IOException {
       return;
     }
 
-    DataInputBuffer in = new DataInputBuffer();
-    in.reset(serializedMetrics, 0, serializedMetrics.length);
-    ScanMetrics scanMetrics = new ScanMetrics();
-    scanMetrics.readFields(in);
-    MetricsTimeVaryingLong[] mlvs = scanMetrics.getMetricsTimeVaryingLongArray();
-
+    ScanMetrics scanMetrics = scan.getScanMetrics();
```

Comment:
what's difference between "new ScanMetics().readFileds(dataInputBuffeer)" and "scan.getScanMetrics()"